dask/dataframe/optimize.py

```diff
@@ -27,9 +27,9 @@ def optimize(dsk, keys, **kwargs):
         dsk, dependencies = cull(dsk, [keys])
     dsk = fuse_getitem(dsk, dataframe_from_ctable, 3)
     if fastparquet:
-        from .io.parquet import _read_parquet_row_group
-
-        dsk = fuse_getitem(dsk, _read_parquet_row_group, 4)
+        #from .io.parquet.fastparquet import _read_parquet_row_group
+        #dsk = fuse_getitem(dsk, _read_parquet_row_group, 4)
+        pass  # TODO: in abeyance
```

Comment:
Would really like to see this come back! This is, I think, the most urgent thing after this PR.

Reply:
@martindurant @mrocklin As far as I can tell, this change is causing a failure in `dask/dataframe/tests/test_shuffle.py::test_temporary_directory` (but only in the [windows appvayor check](https://ci.appveyor.com/project/daskdev/dask/builds/25939563)).  I do not personally understand exactly how this operation fusion works in dask, or why this would lead to a test failure - Any thoughts/suggestions?